src/PKSim.R/Bootstrap/ApplicationStartup.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Globalization;
+using System.IO;
+using System.Reflection;
+using System.Threading;
+using OSPSuite.Core.Services;
+using OSPSuite.Utility.Container;
+using OSPSuite.Utility.Events;
+using OSPSuite.Utility.Exceptions;
+using PKSim.CLI.Core.MinimalImplementations;
+using PKSim.Core;
+using PKSim.Infrastructure;
+
+namespace PKSim.R.Bootstrap
+{
+   internal class ApplicationStartup
+   {
+      private static IContainer _container;
+
+      public static IContainer Initialize()
+      {
+         if (_container != null)
+            return _container;
+
+         redirectNHibernateAssembly();
+
+         _container = new ApplicationStartup().performInitialization();
+         return _container;
+      }
+
+      private IContainer performInitialization()
+      {
+         var container = InfrastructureRegister.Initialize();
+
+         using (container.OptimizeDependencyResolution())
+         {
+            container.RegisterImplementationOf(new SynchronizationContext());
+            container.AddRegister(x => x.FromType<RRegister>());
+            container.AddRegister(x => x.FromType<CoreRegister>());
+            container.AddRegister(x => x.FromType<InfrastructureRegister>());
+            InfrastructureRegister.RegisterSerializationDependencies(container);
+            registerMinimalImplementations(container);
+         }
+
+         var configuration = container.Resolve<IPKSimConfiguration>();
+         configuration.PKSimDbPath = Path.Combine(new FileInfo(Assembly.GetAssembly(GetType()).Location).DirectoryName, CoreConstants.PK_SIM_DB_FILE);
```

Comment:
I have tried a milllion different ways but in short, starting PKSim.R from rClr returns the wrong local path so I have to overwrite it here